core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -21,186 +22,17 @@
  * @since 4.0
  */
 public interface RpcManager {
```

Comment:
More and more I am seeing need for individual rpcs.  I wonder if we should finally add method(s) for this use case?  Something like ```CompletableFuture<Response> invokeRemotely(Address, ...)```